src/devtools/client/debugger/src/components/SecondaryPanes/Frames/NewFrames.tsx

```diff
@@ -64,30 +65,25 @@ function FramesRenderer({
     asyncIndex,
     focusWindow
   );
-  if (asyncParentPauseId === null) {
+  if (asyncParentPauseId === true) {
     return (
       <>
         {asyncSeparator}
         <div className="pane-info empty">
-          This part of the call stack is unavailable.
-          {isFocusWindowApplied(focusWindow, endpoint) && (
-            <>
-              {" "}
-              Perhaps it is outside of{" "}
-              <span className="cursor-pointer underline" onClick={() => dispatch(enterFocusMode())}>
-                your debugging window
-              </span>
-              .
-            </>
-          )}
+          This part of the call stack is unavailable because it is outside{" "}
+          <span className="cursor-pointer underline" onClick={() => dispatch(enterFocusMode())}>
+            your debugging window
+          </span>
+          .
```

Comment:
I think we need to re-add the messaging changes from #9368. We should never show this message if the focus window is the full recording.

Reply:
> Unless you're 100% confident that we'll never have a false positive here

Yes, I'm confident (especially after changing it to *not* return true if `focusWindow` is `null`, as suggested below).